.gitignore

```diff
@@ -1,11 +1,11 @@
 *.pyc
-_trial_temp
-build
+/_trial_temp/
```

Comment:
Since we're being pedantic here, shouldn't this line be `/_trial_temp*/`? Not that this directory will ever exist if you are running the tests via Tox. :man_shrugging: